readdy/test/TestReactions.cpp

```diff
@@ -192,9 +192,41 @@ TEST_P(TestReactions, FissionNearBoundary) {
     const auto &pos1 = particles[0].getPos();
     const auto &pos2 = particles[1].getPos();
     if (pos1.x <= 0.) {
-        EXPECT_GT(pos2.x, 0.);
+        EXPECT_GT(pos2.x, 0.) << "wrong position in " << reactionHandler;
     } else if (pos1.x >= 0.) {
-        EXPECT_LT(pos2.x, 0.);
+        EXPECT_LT(pos2.x, 0.) << "wrong position in " << reactionHandler;
+    }
+};
+
+std::vector<std::string> reactionHandlers = {"UncontrolledApproximation", "Gillespie", "DetailedBalance"};
```

Comment:
I think it would make more sense to add this to the testing matrix (ie two parameters instead of just the kernel)